cloudferry/model/__init__.py

```diff
@@ -15,26 +15,9 @@
 Model module contain tools to define cloud object schema and a way to store
```

Comment:
Can you explain why? I think that it's better to have several {{__init__.py}} is better then several base modules. Reason: probability of error is less when you don't have modules with same names in different packages.